lib/patches/active_record/relation.rb

```diff
@@ -0,0 +1,12 @@
+unless ::ActiveRecord::VERSION::STRING < "5.0.0"
+  module Globalize
+    module Relation
+      def where_values_hash(relation_table_name = table_name)
+        return super unless respond_to?(:translations_table_name)
+        super.merge(super(translations_table_name))
+      end
+    end
+  end
+
+  ActiveRecord::Relation.send(:prepend, Globalize::Relation)
```

Comment:
I don't think prepend is private anymore on the versions of Ruby that Rails 5 supports?